static/app/components/multiPlatformPicker.tsx

```diff
@@ -0,0 +1,300 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import debounce from 'lodash/debounce';
+import {PlatformIcon} from 'platformicons';
+
+import Button from 'sentry/components/button';
+import ExternalLink from 'sentry/components/links/externalLink';
+import ListLink from 'sentry/components/links/listLink';
+import NavTabs from 'sentry/components/navTabs';
+import {DEFAULT_DEBOUNCE_DURATION} from 'sentry/constants';
+import categoryList, {
+  filterAliases,
+  PlatformKey,
+  popular,
+} from 'sentry/data/platformCategories';
+import platforms from 'sentry/data/platforms';
+import {IconClose, IconProject, IconSearch} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {inputStyles} from 'sentry/styles/input';
+import space from 'sentry/styles/space';
+import {Organization, PlatformIntegration} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import EmptyMessage from 'sentry/views/settings/components/emptyMessage';
+
+const PLATFORM_CATEGORIES = [{id: 'all', name: t('All')}, ...categoryList] as const;
+
+const isPopular = (platform: PlatformIntegration) =>
+  popular.includes(platform.id as typeof popular[number]);
+
+const PlatformList = styled('div')`
+  display: grid;
+  gap: ${space(1)};
+  grid-template-columns: repeat(auto-fill, 112px);
+  margin-bottom: ${space(2)};
+`;
+
+type Category = typeof PLATFORM_CATEGORIES[number]['id'];
+
+interface PlatformPickerProps {
+  addPlatform: (key: PlatformKey) => void;
+  organization: Organization;
+  platforms: PlatformKey[];
+  removePlatform: (key: PlatformKey) => void;
+  defaultCategory?: Category;
```

Comment:
Did we end up deciding to include the tab bar anyways

Reply:
@Neo-Zhixing yes